site/content/entry/compatability-list-wiki-created.md

```diff
@@ -0,0 +1,15 @@
++++
+date = "2017-06-03T11:30:00-04:00"
+title = "Compatibility List/Wiki Created!"
+author = "Saphiresurf"
+forum = 1603
++++
+
+[Check it Out Here] (https://citra-emu.org/game/)
```

Comment:
I was thinking since the subject itself was relatively self explanatory that a link at the top could be beneficial for those who want to view it without having to find it, but you do have a point in that it could easily be mistaken for a subtitle.

Also of course the glaring issue that yeah the preview does sort of mush them together so I'll make sure to fix this haha.